src/navigator_data_ingest/base/actions.py

```diff
@@ -146,14 +153,22 @@ def _handle_document(
             document_bucket,
         )
 
-        # TODO: (BAK-1208) Send updated md5sum/url details to API endpoint
-        # update_document_response = post_update(session=session, document=document)
-        # if update_document_response.status_code >= 300:
-        #     # TODO: More nuanced status response handling
-        #     _LOGGER.error(
-        #         f"Failed to update entry in the database for '{document.import_id}': "
-        #         f"{update_document_response.text}"
-        #     )
+        response = update_document_details(
+            session,
+            token,
+            document.import_id,
+            uploaded_document_result,
+        )
+        # FIXME: Remove the debug and test with corresponding backend change
+        print("-" * 100)
+
+        if response.status_code >= 300:
+            # TODO: More nuanced status response handling
+            _LOGGER.error(
+                f"Failed to update entry in the database for '{document.import_id}': "
+                f"[{response.status_code}] {response.text}"
+            )
```

Comment:
is this currently highlighted because we always fail?

Reply:
What do you mean by "highlighted" ?

Reply:
I just meant these extra long print/log separators around the code blocks